.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>benchflow-spark-tasks-sender</name>
+	<name>Spark-tasks-sender</name>
```

Comment:
Most of the others didn't have benchflow in their name. I don't think you need to specify its for benchflow, since it's already in the benchflow repo anyways.